src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.setColor(color);
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.round(Math.PI * radius * radius);
```

Comment:
we should not modify result value, it can be modified somewhere outside, or else we will get not accurate values
```suggestion
        return Math.PI * radius * radius;
```